templates/base.html

```diff
@@ -8,7 +8,12 @@
     <title>{% block title %}AcousticBrainz{% endblock %}</title>
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <link rel="icon" href="{{ url_for('static', filename='img/favicon.png') }}" type="image/x-icon" />
+
+    <link rel="icon" sizes="16x16" href="{{ url_for('static', filename='img/favicon-16.png') }}" type="image/png" />
```

Comment:
You actually use devices that uses favicons? Wow, so modern. :)

more seriously, the list of actual favicons that should be supported to make everything in the world happy is totally redonkulous. I think this is a fair balance.